src/go/rpk/pkg/cli/cmd/cluster/config/import.go

```diff
@@ -110,13 +119,16 @@ func importConfig(
 				upsert[k] = v
 			}
 		} else {
-			// Present in input but not original config, insert
-			upsert[k] = v
-			propertyDeltas = append(propertyDeltas, propertyDelta{k, "", fmt.Sprintf("%v", v)})
+			// Present in input but not original config, insert if it differs
+			// from the materialized current value (which may be a default)
+			if !haveOldValMaterialized || !reflect.DeepEqual(oldValMaterialized, v) {
+				upsert[k] = v
+				propertyDeltas = append(propertyDeltas, propertyDelta{k, "", fmt.Sprintf("%v", v)})
+			}
 		}
 	}
 
-	for k := range oldConfig {
+	for k := range oldConfigFull {
```

Comment:
Line 146 uses `oldConfig` still, which I think given this PR, could not exist if it is a default value?